openapi_python_client/parser/properties/union.py

```diff
@@ -24,6 +53,7 @@ class UnionProperty(PropertyProtocol):
     description: str | None
     example: str | None
     inner_properties: list[PropertyProtocol]
+    discriminators: list[DiscriminatorDefinition] | None = None
```

Comment:
Just want to confirm that this means that the discriminator logic defined here will automatically work for every where polymorphism is supported in a spec including in the request/response bodies?

Reply:
Yes. There isn't separate deserialization logic for request vs. response bodies; they just call `from_dict` on the model class.